pytest.ini

```diff
@@ -18,14 +21,11 @@ filterwarnings=
 	# realpython/pytest-mypy#152
 	ignore:'encoding' argument not specified::pytest_mypy
 
-	# python/cpython#100750
-	ignore:'encoding' argument not specified::platform
-
-	# pypa/build#615
-	ignore:'encoding' argument not specified::build.env
+	# pytest-dev/pytest # TODO: Raise issue upstream 
```

Comment:
@Avasam you added a TODO here, do you plan to open the issue?

I don't know if the `pytest` folks will decide to address this... It seems to be related to the `_py` library embedded in `pytest`, I don't know to which extent that is actively maintained.

Reply:
I need to retrigger the error first, but yes, let's open an issue upstream. Even if we get a "won't fix", at least we can reference that.

Reply:
Otherwise my idea was to mention this: https://github.com/pypa/setuptools/issues/4326

Reply:
I found we can actually fix those on setuptool's side. We're not passing any encoding to pytest's `LocalPath.open` (often the `tmpdir` parameter of a test)